validphys2/src/validphys/commondataparser.py

```diff
@@ -41,36 +41,21 @@
 import logging
 from operator import attrgetter
 from pathlib import Path
-from typing import Any, Dict, Optional
+from typing import Any, Optional
 
 import numpy as np
 import pandas as pd
+from ruamel.yaml import YAML
 from validobj import ValidationError, parse_input
 from validobj.custom import Parser
 
 from nnpdf_data import new_to_legacy_map, path_commondata
 from nnpdf_data.utils import parse_yaml_inp
-
-# We cannot use ruamel directly due to the ambiguity ruamel.yaml / ruamel_yaml
-# of some versions which are pinned in some of the conda packages we use...
-from reportengine.compat import yaml
 from validphys.coredata import KIN_NAMES, CommonData
 from validphys.plotoptions.plottingoptions import PlottingOptions, labeler_functions
 from validphys.process_options import ValidProcess
 
-try:
-    # If libyaml is available, use the C loader to speed up some of the read
-    # https://pyyaml.org/wiki/LibYAML
-    # libyaml is available for most linux distributions
-    Loader = yaml.CLoader
-except AttributeError:
-    # fallback to the slow loader
-    Loader = yaml.Loader
-
-
-def _quick_yaml_load(filepath):
-    return yaml.load(filepath.read_text(encoding="utf-8"), Loader=Loader)
-
+yaml = YAML(typ='rt')
```

Comment:
```suggestion
yaml = YAML(typ='safe')
```

Just benchmarked it, with this change is (almost) as fast as master. Not as fast though, it takes 1.3x more. 
With `rt` it takes 10x more.

As I said, I think the best thing at this point is to have in utils `safe_load` and `rt_load` or whatever with a comment explaining why. That will make it easier if we need to change it again.